experimental/packages/otlp-exporter-base/src/transport/xhr-transport.ts

```diff
@@ -49,8 +49,8 @@ class XhrTransport implements IExporterTransport {
 
       xhr.ontimeout = _ => {
         resolve({
-          status: 'failure',
-          error: new Error('XHR request timed out'),
+          status: 'retryable',
+          retryInMillis: 0,
```

Comment:
this timeout here is also the full maximum of the request - if this is ever triggered, there's no time left for the export to retry.